packages/cursorless-engine/src/actions/Replace.ts

```diff
@@ -63,33 +63,41 @@ export default class Replace {
     await runForEachEditor(
       edits,
       (edit) => edit.editor,
-      async (editor, edits) => {
-        const contentSelections = {
-          selections: edits.map(({ target }) => target.contentSelection),
-        };
-        const editSelections = {
-          selections: edits.map(({ edit }) => edit.range.toSelection(false)),
-          rangeBehavior: RangeExpansionBehavior.openOpen,
-        };
+      async (editor, editWrappers) => {
```

Comment:
whoah this `editWrapper` thing is awkward. I don't love the name `editWrapper` but I agree not sure what to call it. A bigger refactor is prob in order here in the future

Reply:
Totally agree